packages/dashboard-frontend/src/store/FactoryResolver/getDevfile.ts

```diff
@@ -72,7 +76,10 @@ export function getDevfile(data: FactoryResolver, location: string): api.che.wor
     if (!metadata.attributes) {
       metadata.attributes = {};
     }
-    metadata.attributes['dw.metadata.annotations'] = { 'che.eclipse.org/devfile-source': devfileSource };
+    if (!metadata.attributes[DEVWORKSPACE_METADATA_ANNOTATION]) {
+      metadata.attributes[DEVWORKSPACE_METADATA_ANNOTATION] = {};
+    }
+    metadata.attributes[DEVWORKSPACE_METADATA_ANNOTATION][DEVWORKSPACE_DEVFILE_SOURCE] = devfileSource;
```

Comment:
I'm not sure what it's related to but after I accepted https://che-eclipse-che.apps.cluster-bd6a.bd6a.sandbox1891.opentlc.com/f?url=https://github.com/l0rd/spring-petclinic/tree/devfile2&policies.create=peruser I got in the annotations 
```yaml
      factory:
        params: >-
          url=https://github.com/l0rd/spring-petclinic/tree/devfile2?policies.create=peruser
```
but I would expect 
```yaml
      factory:
        params: >-
          url=https://github.com/l0rd/spring-petclinic/tree/devfile2&policies.create=peruser
```
See screenshot
![Screenshot_20210714_160716](https://user-images.githubusercontent.com/5887312/125627370-0e789569-ad2f-414b-af38-f8e9cf233360.png)

Reply:
I have fixed it